### DISCUSSION TITLE
Support `query` for Sentry issues source

### DISCUSSION THREAD

**Discussion by CircleCode** [airbytehq/airbyte#36218] [Connector Ideas and Features]
According to https://docs.sentry.io/api/events/list-a-projects-issues/, sentry API supports a `query` query parameter.
It would for example allow to retrieve issues only for specific environments (note that with current sentry source connector, there is no way at all to get the environment an issue is linked to as this is not returned by the targeted API)

allowing to pass a custom query would allow for example `env:[prod,production]`

**Comment by joel1di1** [comment]
I have the same issue.
I thought of a resolution (tests are missing) :
https://github.com/airbytehq/airbyte/compare/master...joel1di1:airbyte:add-issue-query

[a comment by an automated account is not shown]
